internal/provider/data_source_workspace_outputs.go

```diff
@@ -137,17 +156,29 @@ func (t workspaceOutputsDataSource) Read(ctx context.Context,
 	}
 
 	if workspace == nil {
+		identifier := "unknown"
```

Comment:
You shouldn't need to set the identifier to "unknown" since either the ID or Path will always be set